ui-tests/src/test/java/io/syndesis/qe/pages/integrations/detail/IntegrationDetailPage.java

```diff
@@ -0,0 +1,65 @@
+package io.syndesis.qe.pages.integrations.detail;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+
+public class IntegrationDetailPage extends SyndesisPageObject {
+	
+	public final List<String> actionsSet = new ArrayList<String>();
+
+	private static final class Element {
+		public static final By ROOT = By.cssSelector("syndesis-integration-detail-page");
+
+		public static final By STATUS = By.cssSelector("syndesis-integration-status");
+		public static final By TITLE = By.cssSelector("h1");
+	}
+
+	public SelenideElement getRootElement() {
+		SelenideElement elementRoot = $(Element.ROOT).shouldBe(visible);
+		return elementRoot;
+	}
+
+	public boolean validate() {
+		return getRootElement().is(visible);
+	}
+
+	public String getIntegrationName() {
+		return this.getElementText(Element.TITLE);
+	}
+
+	public void deleteIntegration() {
+		this.clickButton("Delete Integration");
```

Comment:
Shouldn't we rather reuse that nice enum like class `IntegrationDetailActions`?

Reply:
Good idea, done.